### ISSUE TITLE
The package 'A File Icon' is not available. Any alternatives?

### ISSUE THREAD

**Issue by nocturnesound** [dempfi/ayu#200]
Hi, thanks for an amazing theme!
Recently an author "A File Icon" just delete a repository. What alternative to that package can I use?

**Comment by dempfi** [comment]
Wow, this sucks. I'll bring back original ayu icons to fix this mess. Need some time tho. Thx for heads up @nocturnesound

**Comment by nocturnesound** [comment]
Ok, thank you!

**Comment by ardrigh** [comment]
There is a different FileIcons package, is that is a usable alternative?

https://github.com/braver/FileIcons
https://packagecontrol.io/packages/FileIcons

Hopefully themes can use that to replace "A File Icon".

**Comment by ForWhomTheBellTolls** [comment]
Still not fixed?

**Comment by gene1wood** [comment]
The error message given upon installing `ayu` is

```
Package Control

The package 'A File Icon' is not available
```

**Comment by thatcoderagain** [comment]
@gene1wood I'm facing the same error. Anyone know how to fix it. 

An error prompt comes up every time I open sublime text which says.
```
Package Control

The package 'A File Icon' is not available
```

I have tried to open settings(Prefernces.sublime-setting) file and found 
        ```
"ignored_packages":
	[
		"A File Icon",
		"Vintage"
	],
```
So, I deleted "A File Icon" from there and it doesn't matter I keep it or delete it. It comes back every time I close and open Sublime text again.

Response If anyone has the solution to this.

![capture](https://user-images.githubusercontent.com/11584540/50732697-d6a62700-11a5-11e9-90a3-4d7e7dc68fcc.PNG)
